website/docs/r/snmp_user.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Fabric Policies"
+layout: "aci"
+page_title: "ACI: aci_snmp_user"
+sidebar_current: "docs-aci-resource-snmp_user"
+description: |-
+  Manages ACI SNMP User
+---
+
+# aci_user_profile #
+
+Manages ACI SNMP User
+
+## API Information ##
+
+* `Class` - snmpUserP
+* `Distinguished Name` - uni/fabric/snmppol-{snmp_policy_name}/user-{name}
+
+## GUI Information ##
+
+* `Location` - Fabric > Fabric Policies > Policies > Pod > SNMP > {snmp_policy} > SNMP V3 Users
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_snmp_user" "example" {
+  snmp_policy_dn      = "uni/fabric/snmppol-default"
+  name                = "example"
+  authorization_key   = "my_authorization_key"
+  authorization_type  = "hmac-sha1-96"
+  privacy_key         = "my_privacy_key"
+  privacy_type        = "none"
+}
+```
+
+## Argument Reference ##
+
+* `snmp_policy_dn` - (Required) Distinguished name of the parent SNMP Policy object. Type: String.
+* `name` - (Required) Name of the SNMP User object. Type: String.
+* `annotation` - (Optional) Annotation of the SNMP User object. Type: String.
+* `name_alias` - (Optional) Name Alias of the SNMP User object. Type: String.
+* `authorization_key` - (Required) Authorization Key. The authorization key for the SNMP User. The key can be any case-sensitive alphanumeric string up to 64 chars. Type: String.
```

Comment:
Remove -> `Authorization Key.` from `Authorization Key. The authorization key for the SNMP User.` because it is repetitive